x-pack/plugins/siem/public/components/charts/common.tsx

```diff
@@ -79,3 +81,26 @@ export const getSeriesStyle = (
 
   return customSeriesColors;
 };
+
+// Apply margins and paddings: https://ela.st/charts-spacing
```

Comment:
Adding a reference here to show an example from elastic-charts.
https://ela.st/charts-spacing